kinematics_base_test/src/test_kinematics_plugin.cpp

```diff
@@ -340,6 +346,7 @@ TEST(IKFastPlugin, searchIKWithCallback)
     // check height
     if (poses[0].position.z <= 0.0f)
```

Comment:
I can't recall why the height has to be checked here ...

Reply:
I think you made this in order to check if the IKCallback works.